package.xml

```diff
@@ -4,10 +4,13 @@
   <name>rosidl_runtime_py</name>
   <version>0.9.1</version>
   <description>Runtime utilities for working with generated ROS interfaces in Python.</description>
-  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
+
+  <maintainer email="shane@openrobotics.org">Shane Loretz</maintainer>
```

Comment:
```suggestion
  <maintainer email="sloretz@openrobotics.org">Shane Loretz</maintainer>
```

Reply:
Whoops, I did select that email on purpose because I saw that `shane@` and `sloretz@` were both used to list you as a maintainer on several packages and I thought `shane@` was more consistent with most of the teams' emails (just first name). Looking at it now, `shane@` might have been an error. Here's one example: https://github.com/ros2/system_tests/blob/519e6491fa3ecb1c60edeabc5c23364e56e30140/test_cli_remapping/package.xml.